recipes/rsirius/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "0.9" %}
+{% set name = "r-sirius-ms" %}
+{% set commit = "d1274f51b2af44c09def72fa3628f82a9e8de6b5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/boecker-lab/sirius-client-openAPI/archive/{{ commit }}.zip
+  sha256: 1edef52c472b9984b74532cf41ae9389f4191579085fd981922d8356d0ba887a
+
+build:
+  noarch: generic
+  number: 0
```

These fields are also typically set for R package recipes:

```suggestion
  number: 0
  merge_build_host: True  # [win]
  rpaths:
    - lib/R/lib/
    - lib/
```